discord/ext/commands/context.py

```diff
@@ -322,3 +322,41 @@ async def send_help(self, *args):
                 return None
         except CommandError as e:
             await cmd.on_help_command_error(self, e)
+
+    async def reply(self, content=None, *, mention_author=None, **kwargs):
+        """|coro|
+
+        A shortcut method to :meth:`~discord.abc.Messageable.send` to reply to the
+        :class:`~discord.Message` that invoked the command.
+
+            .. versionadded:: 1.6
+
+        Parameters
+        --------
+        content: Optional[:class:`str`]
+            The content of the message to be sent.
+        mention_author: Optional[:class:`bool`]
+            Shortcut to passing an :class:`AllowedMentions` object with
+            :attr:`AllowedMentions.replied_user` set.
+        kwargs:
+            Other keyword arguments passed to :meth:`abc.Messageable.send`.
+
+        Raises
+        --------
+        ~discord.HTTPException
+            Sending the message failed.
+        ~discord.Forbidden
+            You do not have the proper permissions to send the message.
+        ~discord.InvalidArgument
+            The ``files`` list is not of the appropriate size or
+            you specified both ``file`` and ``files``.
+
+        Returns
+        ---------
+        :class:`~discord.Message`
+            The message that was sent.
+        """
+        allowed_mentions = kwargs.pop('allowed_mentions', discord.AllowedMentions())
+        if mention_author is not None:
+            allowed_mentions.replied_user = mention_author
+        return await self.message.reply(content, allowed_mentions=allowed_mentions, **kwargs)
```

Comment:
Why is this not a duplicate of `Message.reply`?
Would lower redundancy and allow you to do `reply.__doc__ = Message.reply.__doc__`.